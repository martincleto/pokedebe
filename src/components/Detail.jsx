
import React, { Component } from 'react'

//require('stylesheets/detail.scss')

class Detail extends Component {
  constructor(props) {
    super(props)
  }

  render() {
    return (
      <p>Detail</p>
    )
  }
}

export default Detail
