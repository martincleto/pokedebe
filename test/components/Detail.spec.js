import React from 'react'
import renderer from 'react-test-renderer'
import Detail from 'Components/Detail'

describe('Detail', function() {
  test('should render without error', () => {
    const component = renderer.create(
      <Detail/>
    ).toJSON();

    expect(component).toMatchSnapshot();
  });
})
