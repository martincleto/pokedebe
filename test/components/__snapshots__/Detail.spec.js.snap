// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail should render without error 1`] = `
<div
  className="detail__inner-wrapper"
>
  <header>
    <h1
      className="detail__title"
    >
      dragonite
    </h1>
    <p>
      Pokedex #
    </p>
  </header>
  <ul
    className="detail__attributes-list"
  >
    <li
      className="detail__attributes-item"
    >
      Weight: 
      <strong />
    </li>
    <li
      className="detail__attributes-item"
    >
      Height: 
      <strong />
    </li>
  </ul>
  <p
    className="detail__back-button"
  >
    <a
      onClick={[Function]}
      style={Object {}}
    >
      <i
        className="icon-back"
      />
      Back to Search
    </a>
  </p>
</div>
`;
